docs/extras/guide/generated_v1.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Generated API PHP
+---
+The PHP OCR SDK supports generated APIs.
+Generated APIs can theoretically support all APIs in a catch-all generic format.
+
+# Quick-Start
+
+```php
+<?php
+
+use Mindee\Client;
+use Mindee\Product\Generated\GeneratedV1;
+use Mindee\Input\PredictMethodOptions;
+
+// Init a new client
+$mindeeClient = new Client("my-api-key");
+
+// Load a file from disk
+$inputSource = $mindeeClient->sourceFromPath("/path/to/the/file.ext");
+
+// Create a custom endpoint
+$customEndpoint = $mindeeClient->createEndpoint(
+    "my-endpoint",
+    "my-account",
+    "my-version"
+);
+
+// Add the custom endpoint to the prediction options.
+$predictOptions = new PredictMethodOptions();
+$predictOptions->setEndpoint($customEndpoint);
+
+// Parse the file
+$apiResponse = $mindeeClient->enqueueAndParse(GeneratedV1::class, $inputSource, $predictOptions);
+
+echo strval($apiResponse->document);
+```
+
+# Generated Endpoints
+
+You may have noticed in the previous step that in order to access a custom build, you will need to provide an account and an endpoint name at the very least.
```

Comment:
```suggestion
As shown above, you will need to provide an account and an endpoint name at the very least.
```
It's not just for custom endpoints, all endpoints require setting the account and name, no?